server/routes/api.js

```diff
@@ -0,0 +1,90 @@
+
+const express = require('express')
+const router = express.Router()
+const { data, map } = require('jquery')
+const urllib = require('urllib');
+const dreamTeam = []
+
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756"
+}
+
+router.get('/teams/:teamName', function (req, res) {
+    const teamName = req.params.teamName;
+    urllib.request('http://data.nba.net/10s/prod/v1/2018/players.json', function(err, response){
+        const data = JSON.parse(response.toString())
+        const allData = data.league.standard
+        const filteredData = allData.filter(f => f.teamId == teamToIDs[teamName])
+        const finalData = filteredData.map(fd => ({'firstName': fd.firstName, 'lastName': fd.lastName, 'jersey': fd.jersey, 'pos': fd.pos}))
+        res.send(finalData)
+    })
+})
+
+router.get('/playerStats/:player', function (req, res) {
+    const player = req.params.player;
```

Comment:
you can deconstruct objects and it looks a lot better
`const { player } = req.params`
it is the same as 
`const player = req.params.player`